core/src/main/java/com/threerings/getdown/data/Application.java

```diff
@@ -1546,7 +1552,6 @@ protected void downloadControlFile (String path, int sigVersion)
 
                         if (!sig.verify(Base64.decode(signature, Base64.DEFAULT))) {
                             log.info("Signature does not match", "cert", cert.getPublicKey());
-                            continue;
```

Comment:
What is the purpose of changing this code in an otherwise totally unrelated patch?

Similar comments could be made for all the other cosmetic changes, but I will let those slide because I appreciate the goal of keeping the code clean and up to date.

But behavioral changes should at a minimum be in a separate PR, explaining why they were made.

Reply:
totally agree. the things are simple: i'm working in IntelliJ and just hit "Clean Code" in some place. then just run mvn test - commit - push :)